extensions/wikia/UserLogin/js/UserSignup.js

```diff
@@ -125,7 +111,7 @@
 	// expose global
 	window.UserSignup = UserSignup;
 
-	$(window).on('load', function () {
```

Comment:
Since we no longer have to wait for the captcha script to be loaded, we can initialize on `DOMContentLoaded` instead of `window.onload`.